src/commands/mail/create.ts

```diff
@@ -0,0 +1,152 @@
+import ora, { Ora } from 'ora';
+import inquirer from 'inquirer';
+import Command, { IConfig } from '../../base.js';
+import { Flags } from '@oclif/core';
+import {
+  MAILBOX_MODES,
+  MAILBOX_PLANS,
+  REGIONS_API_URL,
+  DEV_MODE,
+} from '../../constants.js';
+import { createDebugLogger } from '../../utils/output.js';
+
+export default class MailCreate extends Command {
+  static description = 'create a MailServer';
+
+  static flags = {
+    ...Command.flags,
+    domain: Flags.string({
+      description: 'domain',
+    }),
+    plan: Flags.string({
+      description: 'plan',
+    }),
+    mode: Flags.string({
+      description: 'mode',
+    }),
+  };
+
+  static aliases = ['mail:create'];
+
+  spinner!: Ora;
+
+  async setGotConfig(config: IConfig): Promise<void> {
+    await super.setGotConfig(config);
+    this.got = this.got.extend({
+      prefixUrl: DEV_MODE ? 'http://localhost:3000' : REGIONS_API_URL['mail'],
```

Comment:
And also please move this to the constants file.

```suggestion
      prefixUrl: DEV_MODE ? 'http://localhost:3002' : REGIONS_API_URL['mail'],
```